homeassistant/components/fitbit/sensor.py

```diff
@@ -165,7 +165,7 @@ def setup_platform(
         request_app_setup(hass, config, add_entities, config_path, discovery_info=None)
         return
 
-    if "fitbit" in _CONFIGURING:
+    if _CONFIGURING.get("fitbit") is not None:
```

Comment:
Why?

Reply:
I thought this is more correct written :thinking:

Reply:
Old one is much cleaner.

Reply:
ah ok ... i always assumed that `xxx.get() is not None` is the more common

Reply:
I agree with with KapJI, but do not forget that .get allows a default value:
  _CONFIGURING.get("fitbit", myFibit) 
That can save a lot of “if”.